daq/host.py

```diff
@@ -602,6 +641,7 @@ def _get_switch_config(self):
         return {
             'ip': self.switch_setup.get('ip_addr'),
             'model': self.switch_setup.get('model'),
+            'telnet_port': int(self.switch_setup.get('telnet_port', self._DEFAULT_TELNET_PORT)),
```

Comment:
'telnet_port' isn't right -- what is it used for?  usi_port?

Reply:
the telnet port for usi to talk to the switch. usi port is defined in the usi_url config

Reply:
as I mentioned elsewhere "telnet_port" is way to generic and not all that interesting (telnet is a protocol) -- the name should indicate more the use not the protocol.

Reply:
I understand that the _DEFAULT_TELNET_PORT should be more descriptive but the telnet_port is already under the switch_setup config so isn't saying switch_telnet_port redundant?  same with username, password, model etc.